src/ESPEasy.ino

```diff
@@ -818,13 +833,16 @@ void checkSystemTimers()
       if (timeOutReached(systemTimers[x].timer))
       {
         struct EventStruct TempEvent;
+        TempEvent.TaskIndex = systemTimers[x].TaskIndex;
         TempEvent.Par1 = systemTimers[x].Par1;
         TempEvent.Par2 = systemTimers[x].Par2;
         TempEvent.Par3 = systemTimers[x].Par3;
+        TempEvent.Par4 = systemTimers[x].Par4;
+        TempEvent.Par5 = systemTimers[x].Par5;
+        systemTimers[x].timer = 0;
```

Comment:
Is this a proper bug-fix, to set the timer to 0 before calling each plugin_ptr?
It sounds like a proper fix, since the timeout was reached.
Just to make sure it was a deliberate fix.

Reply:
it is intentional